features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -461,6 +508,77 @@ Quite distinctly from the [auto smart crop](resize-crop-and-other-transformation
 **Note:** Smart crop may not give accurate results for some images. This is partially a trade off between speed (needed for real-time transformations) and accuracy.
 {% endhint %}
 
+### Object aware cropping
```

Comment:
This cannot be a level 3 heading. All level 3 headings in this page are in the `${transformation_name} - (${transformation_key})` format.

Make it a level 4 heading under `Focus - (fo)` like auto crop and face crop

Reply:
But it is more relevant ,  keeping at level 4 `fo-auto ` and `fo-face` it will be bit lost case